docs/book/themes/themes.rst

```diff
@@ -33,6 +33,7 @@ To use themes inside of your project you need to add these few lines to your ``c
    sylius_theme:
        sources:
            filesystem:
+               scan_depth: 1
```

Comment:
It is not mandatory, isn't it ?

I mean, this should be part of some config_reference, but not mandatory for installing the themes

Reply:
It's not mandatory. But it's configured by default with the sylius standard and is a nice performance improvement.